package.json

```diff
@@ -25,7 +26,7 @@
     "passport-local-mongoose": "^3.1.0",
     "passport-stub": "^1.0.1",
     "serve-favicon": "~2.3.0",
-    "twilio": "^2.8.0"
+    "twilio": "^3.0.0-rc.13"
```

Comment:
Would you mind using the strict version of this dependency?

`^3.0.0-rc.13` actually means `>=^3.0.0-rc.13 and <=4.0.0` -- I don't think we'd like to introduce such level of compatibility :cold_sweat:. If you want to talk about it please ping me.